components/yaml.rst

```diff
@@ -120,11 +115,26 @@ error occurred:
     use Symfony\Component\Yaml\Exception\ParseException;
 
     try {
-        $value = Yaml::parse(file_get_contents('/path/to/file.yml'));
+        $value = Yaml::parse('...');
     } catch (ParseException $e) {
-        printf("Unable to parse the YAML string: %s", $e->getMessage());
+        printf('Unable to parse the YAML string: %s', $e->getMessage());
     }
 
+Reading YAML Files
+~~~~~~~~~~~~~~~~~~
+
+The :method:`Symfony\\Component\\Yaml\\Yaml::parseFile` method parses the YAML
+contents of the given file path and converts them to a PHP value::
+
+    use Symfony\Component\Yaml\Yaml;
+
+    $value = Yaml::parseFile('/path/to/file.yml');
```

Comment:
The method is not static https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Yaml/Parser.php#L49

Reply:
but it is static in the `Yaml` class (which is used here): https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Yaml/Yaml.php#L53 :)

Reply:
Ah, you're right!